src/FormNetworks/bilinearformnetwork.jl

```diff
@@ -0,0 +1,62 @@
+default_bra_vertex_map(v) = (v, "bra")
+default_ket_vertex_map(v) = (v, "ket")
+default_operator_vertex_map(v) = (v, "operator")
+default_operator_constructor(s::IndsNetwork) = delta_network(s)
+
+struct BilinearFormNetwork{V,KetMap,BraMap,OperatorMap} <: AbstractFormNetwork{V}
+  tn::AbstractITensorNetwork{V}
```

Comment:
```suggestion
struct BilinearFormNetwork{V,TensorNetwork<:AbstractITensorNetwork{V}
,BraMap,KetMap,OperatorMap} <: AbstractFormNetwork{V}
  tensornetwork::TensorNetwork
```
We should avoid having fields with abtract types, see https://docs.julialang.org/en/v1/manual/performance-tips/#Avoid-fields-with-abstract-type.